packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -2239,13 +2206,6 @@ def latest_result(self) -> Optional[BaseSynchronizedData]:
         """Get the latest result of the round."""
         return None if len(self._round_results) == 0 else self._round_results[-1]
 
-    @property
-    def background_round_tx_type(self) -> Optional[str]:
-        """Returns the allowed transaction type for background round."""
-        if self.is_termination_set:
-            return cast(AppState, self.background_round_cls).allowed_tx_type
-        return None  # pragma: no cover
```

Comment:
typing nightmare with the payload_class instead of tx type. Because returning `payload_class:  BaseTxType` does not allow for comparison with ininstance when None is returned (since nothing is an instance of `None`). Anyway I believe it doesn't make any sense to expose, not in the first place, and certainly not now that tx type is removed.

Reply:
Ok I see we do this in-place down below